site/docs/Contributing/design.mdx

```diff
@@ -0,0 +1,248 @@
+  - ❇️ **[SECTION NAME]** can be used to highlight _sections_ (for example `❇️ NAVIGATION`).
+  - 🔹**[Item]** can be used to highlight _overridable items_. The name can also be in _all caps_ to make it more distinct (for example `🔹 Base - link`, `🔹DIVIDER`).
+- **Sketch smart layout hacks** may demand extraneous groups to work. In these situations, mark `(smart layout)` in the name of the group.
+
+> **Example: Emoji prefixes in *HDS Footer* layer names**
+> <Image src="../../static/contributing-sketch-layer-prefix.png" alt="Emoji prefixes in HDS Footer layer names" style="max-width:230px" viewable />
+
+
+### Naming variant types
+- **The basic form** of the component can be named `01 Basic`
+- **Recommended** variant can be named `01 Default`
+- **Size** variants are named with _T-shirt sizes_ `S`, `M`, `L`, `XL`, `2XL`…).
+- **Colour theme** variants are named `01 light` and `02 Dark` according to their background colour. 
+- **breakpoints** are always named in the same way:
+  - `01 XL Desktop (≥ 1248)`
+  - `02 L Desktop (≥ 992)`
+  - `03 M Tablet (≥ 768)`
+  - `04 S Mobile (≥ 576)`
+  - `05 XS Mobile (≥ 320)`
+
+> **Example: Colour theme and breakpoint variants in HDS Footer**
+> <Image src="../../static/contributing-sketch-symbol-variants.png" alt="Colour theme and breakpoint variants of footer" style="max-width:230px" viewable />
+
+
+### Sorting variants
+Numbering can be used in front of the name for listing symbols in specific order.
+- **Running numbering** with a leading zero is used for general sorting (for example `01 Primary`, `02 Secondary`, `03 Supplementary`).
+- **Semantic numbering** can be used to help communicating variants (for example naming font weights by css values: `400 Regular`, `500 Medium`, `700 Bold`.
+- A good practice is to order variants from simple to complex (for example `01 Text`, `02 Text + heading`…).
+
+> **Example: Sorting variants in *HDS Typography* uses custom prefix for alignment**
+> <Image src="../../static/contributing-sketch-symbol-sorting.png" alt="Sorting variants in HDS Tpography" style="max-width:180px" viewable />
+
+
+---
+
+## Quality assurance checklist 
+With the following checklist you can check your design files before merging them to HDS master.
+
+Besides checking HDS library files, this list can also be applied to any Sketch project for better organisation.
+
+<p>
+    This checklist is (in most parts) also available as a <Link href="https://www.sketch.com/extensions/assistants/" external>Sketch Assistant</Link>, that automatically runs a preflight check for Sketch files.
+</p> 
+<p> 
+    For more informationb see the <Link href="https://github.com/ronijaakkola/hds-assistant" external>HDS Assistant Github page</Link>. You can also <Link href="sketch://add-assistant?url=https://github.com/ronijaakkola/hds-assistant/releases/latest/download/hds-assistant.tgz" external>download and install the HDS assistant</Link> directly to currently open Sketch file.
```

Comment:
> download and install the HDS assistant

Laittaisin "add HDS Assistant directly to Sketch via pressing this link" - tai jotain vastaavaa.